src/pages/Main/Main.js

```diff
@@ -1,13 +1,47 @@
 import React from 'react';
 import MainSection from './Components/MainSection/MainSection';
 import './Main.scss';
-
 export default class Main extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      movieInformationList: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/MainMockData.json', {
```

Comment:
```suggestion
    fetch('/data/MainMockData.json', {
```
로컬에 접속할때는 앞에 base url 작성 안해주셔도 됩니다!

Reply:
깜박하고 있었네요 수정했습니다!